deploy-as-code/helm/environments/mgramseva-qa.yaml

```diff
@@ -221,6 +221,7 @@ cluster-configs:
         redis-host: "redis.mgramseva"
         adapter-master-data-service: "http://adapter-master-data-service.mgramseva:8080/"
         ifix-department-entity-service: "http://ifix-department-entity-service.mgramseva:8080/"
+        mgramseva-service: "https://mgramseva-qa.egov.org.in"
```

Comment:
Don't write this as `mgramseva-service`. mGramSeva is not a single service, it is a cluster/system. 
Write the key as `mgramseva-domain-name`. (Or you could just reuse the variable present on line 100, but its better to define this separately. )

Reply:
done

Reply:
@rushang7-eGov and @rahu-eGov the domain for mGramseva is already defined in this file. Please check.